Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
I think it's an excellent idea :)

Also, while I'm thinking about pet peeves, what in the world is the rule concerning 

```
desc "blah"
etc. etc.

head "https://foourl"
```
OR
```
desc "blah"
etc. etc.
head "https://foourl"
```

and whether there should be a blank line above `head ...` or not?

There seems to be a silent war going on between the blank line faction and the no blank line faction.